CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required (VERSION 2.6)
```

Comment:
Let's bump this up to 3.2 and make sure that QA environments have a recent version installed. Under Windows, we'd either build directly from the VC++ project or have a recent cmake version available anyway, and on Travis we can make sure it's at least 3.2, so we might as well take advantage of the nicer syntax that we'd get with 3.0+. Since this is an internal project I'm not too bothered about supporting old platforms!